bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/util/HexUtils.java

```diff
@@ -0,0 +1,56 @@
+/**
+ * Copyright (c) 2014,2018 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.eclipse.smarthome.core.util;
+
+/**
+ * Static utility methods that are helpful when dealing with hex data and byte arrays.
+ *
+ * @author Kai Kreuzer - Initial contribution and API
+ *
+ */
+public class HexUtils {
```

Comment:
Such stuff IMHO should also have test-cases, primarily to prevent accidental regressions.